src/rules.mk

```diff
@@ -98,6 +98,12 @@ help:
 	@ printf '\033[37m  reset  \033[00m Reset the build\n'
 
 $(ACTIVATE):
+
```

Comment:
@amotl can you land a new PR to remove this whitespace and the white space below? in general, I try to keep make targets compact with no empty command lines. thx!!